packages/cosmwasm-stargate/src/cosmwasmclient.ts

```diff
@@ -166,11 +192,11 @@ export class CosmWasmClient {
   }
 
   public disconnect(): void {
-    this.tmClient.disconnect();
+    this.forceGetTmClient().disconnect();
```

Comment:
Can we unify this with `packages/stargate/src/stargateclient.ts`? There we have 

```ts
  public disconnect(): void {
    if (this.tmClient) this.tmClient.disconnect();
  }
```

for no reason. I think both options are okay.